tests/observation/test_grdc.py

```diff
@@ -78,9 +81,54 @@ def test_get_grdc_data(tmp_path, sample_grdc_file):
             'river_name': 'SOME RIVER',
             'station_name': 'SOME',
             'time_series': '2000-01 - 2000-01',
-            'units': 'm�/s'
+            'units': 'm³/s',
+            'UserEndTime': '2000-02-01T00:00Z',
+            'UserStartTime': '2000-01-01T00:00Z',
+            'nrMissingData': 1,
         }
-    )
+    return data, metadata
+
+
+def test_get_grdc_data_with_datahome(tmp_path, expected_results):
+    expected_data, expected_metadata = expected_results
+    result_data, result_metadata = get_grdc_data('42424242', '2000-01-01T00:00Z', '2000-02-01T00:00Z', data_home=tmp_path)
```

Comment:
tmp_path is Path object and data_home should be str. Either stringify or allow Path in data_home typing
```suggestion
    result_data, result_metadata = get_grdc_data('42424242', '2000-01-01T00:00Z', '2000-02-01T00:00Z', data_home=str(tmp_path))
```